app/src/main/java/com/kabouzeid/gramophone/model/lyrics/Lyrics.java

```diff
@@ -0,0 +1,49 @@
+package com.kabouzeid.gramophone.model.lyrics;
+
+import com.kabouzeid.gramophone.model.Song;
+
+public class Lyrics {
+    public Song song;
+    public String data;
+
+    boolean parsed = false;
+    boolean valid = false;
+
+    public Lyrics(Song song, String data) {
+        this.song = song;
+        this.data = data;
+    }
+
+    public static Lyrics parse(Song song, String data) {
```

Comment:
I'm a bit unsure about specifically using `SynchronizedLyricsLRC` in the base `Lyrics` class. Will it become complicated if/when we add more formats?